src/bin/index.ts

```diff
@@ -50,7 +51,9 @@ export async function bin(args: string[]) {
 		return 1;
 	}
 
-	const { code, options } = await { create, initialize, migrate }[mode](args);
+	const { code, options, zodError } = await { create, initialize, migrate }[
```

Comment:
[Style] As a total non-blocking aside, I went back and forth between making an object like `const runners = { create, ... }` in the root of the file or in the function. It might look cleaner here? Not a blocker whichever way you prefer 😄 just raising that I'm not picky.

Reply:
I think making a separate variable might make it a little bit more readable.